tests/kernel_bundle/get_kernel_bundle_zero_devices.cpp

```diff
@@ -0,0 +1,100 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Check the sycl::get_kernel_bundle logic for the cases when:
+//  - an empty device list was provided
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+#include "get_kernel_bundle.h"
+
+#define TEST_NAME get_kernel_bundle_zero_devices
+
+namespace TEST_NAMESPACE {
+using namespace sycl_cts;
+using namespace sycl_cts::tests::kernel_bundle;
+using namespace sycl_cts::tests::get_kernel_bundle;
+
+template <sycl::bundle_state State>
+void run_tests(util::logger &log, sycl::queue &queue) {
+  const auto context = queue.get_context();
+  const auto device = queue.get_device();
+
+  log.debug([] {
+    return "Running test for " + get_cts_string::for_bundle_state<State>() +
+           " kernel bundle state...";
+  });
+
+  // Ensure we have some kernel in application to retrieve its id
+  using kernelDescriptorT = typename kernels::simple_kernel_descriptor;
+  define_kernel<kernelDescriptorT, State>(queue);
+
+  using kernelNameT = typename kernelDescriptorT::type;
+  const auto kernel_id = sycl::get_kernel_id<kernelNameT>();
+
+  using descriptionT = TestCaseDescription<State>;
+  const std::vector<sycl::device> zero_devices;
```

Comment:
```suggestion
  const std::vector<sycl::device> zero_device;
```

Reply:
In this case using some alghorithm its not so easy, because of we should save `device_img` somewhere.

> Furthermore, as soon as selector_was_called_for_built_in is true, it is no longer required to be computed again in the next selector invocation, so you can guard this with an if (!selector_was_called_for_built_i).

I just break the sycle in case `!device_img.has_kernel(built_in_kernel_id)` is `true`.
I've done it [here](https://github.com/KhronosGroup/SYCL-CTS/pull/224/commits/c151a8cdae55ff2813aa9db1ffde34b33ebd1c11#diff-cfce662345b154bd62edfd6bfceb4dc9e924282661c3234f57122c0262abe876R85)